packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -498,6 +498,7 @@ const conf = (module.exports = convict({
             'https://lockbox.firefox.com/fxa/android-redirect.html',
             'https://accounts.firefox.com/oauth/success/a2270f727f45f648', // Fenix
             'https://accounts.firefox.com/oauth/success/3c49430b43dfba77', // Reference browser
+            'https://accounts.firefox.com/oauth/success/85da77264642d6a1', // Firefox for FireTV
```

Comment:
Having *this* config here and all other config in cloudops-deployment is a recipe for confusion and disaster! What do we need to do to get this out of here and into cloudops-deployment instead?

Reply:
Opened https://github.com/mozilla/fxa/issues/1903